### PR TITLE
`ibm-cloud-text-to-speech` integration params

### PR DESCRIPTION
**Pull request by janhalama**
## Description

This PR adds integration parameters to `ibm-cloud-text-to-speech` provider, it also updates corresponding `speech/synthesis` map, test recordings and snapshots.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by freaz** [comment]
@janhalama when I run locally `yarn capabilities:test capabilities/speech/synthesis/maps/ibm-cloud-text-to-speech.test.ts` tests are not passing.

And same on CI

image

**Review by freaz — CHANGES_REQUESTED** [review]
Tests are not passing

**Comment by janhalama** [comment]
> Tests are not passing

I confirm that when I clone new station and try to run tests they are not passing. When I run tests after recording they pass... I am looking into it.

**Comment by janhalama** [comment]
Tests are not passing because integration parameters are not configured. In case of `ibm-cloud-text-to-speech` provider integration parameters are used to compose url. Testing lib is not matching recordings when integration parameters are not configured or are configured differently.

**Comment by freaz** [comment]
Do you plan to fix it?

**Review by freaz — COMMENTED** [review]
Noticed there are old recordings.json

File: .env.capabilities.example [hunk 1]
```diff
@@ -35,12 +35,14 @@ BITBUCKET_TOKEN=xxx
...
 # PARAMETERS
 
+IBM_CLOUD_TEXT_TO_SPEECH_INSTANCE_ID=xxx
+IBM_CLOUD_TEXT_TO_SPEECH_REGION=eu-de
 MAILGUN_DOMAIN=example.com
 SMARTYSTREETS_AUTH_ID=xxx
 NEXMO_API_SECRET=xxx
 PLIVO_AUTH_ID=xxx
 TWILIO_SID=xxx
+TWILIO_COMMUNICATION_SENDMESSAGE_FROM=+xxx
```
Comment (freaz) [inline .env.capabilities.example:48]: Should this be here?

**Review by freaz — APPROVED** [review]
Looks good

File: .env.capabilities.example (hunk 1 above)
Comment (janhalama) [reply .env.capabilities.example]: Removed. Thanks for catch.

**Pull request merged**